codecs/charLS-FixedMemory-browser.js

```diff
@@ -1,8 +1,11 @@
 /*! CharLS.js - v2.0.1 - 2016-06-08 | (c) 2016 Chris Hafey | https://github.com/chafey/charls */
+
+// Notes:
+// - Had to patch parsejsfunc
```

Comment:
👀

Reply:
Yeah I don't want to be editing these files but this seemed like the shortest-path way to get the functionality working. Once this is all merged in we can update CharLS and OpenJPEG to the latest version and rebuild them with the latest Emscripten stuff. For now this will do.